### ISSUE TITLE
Unable to use Sapi5 with Narrator, the Windows 11 screen reader

### ISSUE THREAD

**Issue by Ambro86** [datajake1999/SAPI5IBMTTS#6]
I successfully installed Sapi5 with the libraries, also using the italian.reg registry file. However, when I try to use text-to-speech with Narrator it says it cannot be used. Could you tell me if there is a way to use it also with Narrator? I have Windows 11 64bit updated to the latest version. Thank you.

**Comment by lbk2907** [comment]
For now has no way. It is because this TTS only support 32 bit. Correct me 
if i'm wrong.

**Comment by titet11** [comment]
this project has been very neglected, and you're right, this project only supports 32-bit programs, that's why it works with NVDA and other text inputs.

This project has a great future, I would love to see them work on it again and add new features.

**Comment by titet11** [comment]
el proyecto está abandonado, lo cual es bastante desepcionante.
